### PR TITLE
Fixes runtime adding/editing laws in law manager

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Fixes a bug where you would be unable to edit the text of laws in the law manager as a borg/AI/admin.

Fixes: #24314
Fixes: #24292

## Why It's Good For The Game
Runtime bad.

## Testing
Be an emagged borg.
Edit a law and add it, it works.

## Changelog
:cl:
fix: Law manager can once again edit and add laws properly.
/:cl:

### CODE REVIEW

**Review by GDNgit — APPROVED** [review]
[oh yeah woo yeah oh yeah woo t](https://www.youtube.com/watch?v=6a-tnmyXjAE)

File: code/modules/tgui/modules/law_manager.dm [hunk 1]
```diff
@@ -69,22 +69,22 @@
 				owner.add_supplied_law(supplied_law_position, supplied_law)
 
 		if("change_zeroth_law")
-			var/new_law = tgui_input_text(src, "Enter new law Zero. Leaving the field blank will cancel the edit.", "Edit Law", zeroth_law)
+			var/new_law = tgui_input_text(usr, "Enter new law Zero. Leaving the field blank will cancel the edit.", "Edit Law", zeroth_law)
```
Comment (Contrabang) [inline code/modules/tgui/modules/law_manager.dm:72]:
```suggestion
			var/new_law = tgui_input_text(ui.user, "Enter new law Zero. Leaving the field blank will cancel the edit.", "Edit Law", zeroth_law)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
These should all be ui.user, which you need to pull from `ui_act(action, list/params, datum/tgui/ui)

**Review by Contrabang — APPROVED** [review]
lgtm

**Review by lewcc — APPROVED** [review]
LGTM

**Pull request merged**